src/engraving/libmscore/marker.cpp

```diff
@@ -56,7 +56,7 @@ const MarkerTypeItem markerTypeTable[] = {
 
 int markerTypeTableSize()
 {
-    return sizeof(markerTypeTable) / sizeof(MarkerTypeItem) - 1; //-1 for the user defined
+    return markerTypeTable.size() - 1; //-1 for the user defined
```

Comment:
Better to add an explicit cast to int here:

return static_cast<int>(markerTypeTable.size()) - 1;

Reply:
Actually, this method is no longer needed. You can remove it and use markerTypeTable.size() instead